web/pdf_print_service.js

```diff
@@ -310,7 +310,8 @@ if ('onbeforeprint' in window) {
 var overlayPromise;
 function ensureOverlay() {
   if (!overlayPromise) {
-    overlayPromise = OverlayManager.register('printServiceOverlay',
+    overlayManager = PDFViewerApplication.overlayManager;
```

Comment:
Given that this file binds itself to `window.print`, I'm wondering if there's any way that a user might be able to trigger printing before `PDFViewerApplication` is completely initialized?

I've not been successful in triggering this myself; but we might want to add an additional check here just for good measure, to avoid an uncaught `Error` if the `overlayManager` isn't defined yet!? Something like this, perhaps:
```javascript
if (!overlayManager) {
  throw new Error('OverlayManager not initialized.'); // Might need a better error message...
}  
```